src/models/transformer.py

```diff
@@ -17,13 +17,12 @@ def __init__(self, backbone: nn.Module, x_dim: int, n_positions: int, n_embd: in
 
     def forward(self, xs, ys):
         self._backbone.to(xs.device) # pyright: ignore[reportArgumentType,reportAttributeAccessIssue]
-        inds = torch.arange(ys.shape[1])
 
         zs = ContextModel.interleave(xs, ys)
         embeds = self._read_in(zs)
         output = self._backbone(inputs_embeds=embeds).last_hidden_state # pyright: ignore[reportCallIssue]
         prediction = self._read_out(output)
-        return prediction[:, ::2, 0][:, inds]  # predict only on xs
+        return prediction[:, ::2, :1] # predict only on xs
 
```

Comment:
is the comment `# predict only on xs` still accurate after we make this change?

Reply:
yes,  the `::2` part selects every other prediction